sdk/identity/identity/integration/AzureKubernetes/Dockerfile

```diff
@@ -2,7 +2,7 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 # ------------------------------------
-ARG NODE_VERSION=20
+ARG NODE_VERSION=18
```

Comment:
doesn't look like the MCR mirror has node 20 alpine. I could use node 20 but it's a much heavier container image